test/test-framework/cmd/manager/main.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"log"
+	"runtime"
+
+	k8sutil "github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	"github.com/operator-framework/operator-sdk/test/test-framework/pkg/apis"
+	"github.com/operator-framework/operator-sdk/test/test-framework/pkg/controller"
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+)
+
+func printVersion() {
+	log.Printf("Go Version: %s", runtime.Version())
+	log.Printf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
+	log.Printf("operator-sdk Version: %v", sdkVersion.Version)
+}
+
+func main() {
+	printVersion()
+	flag.Parse()
+
+	namespace, err := k8sutil.GetWatchNamespace()
+	if err != nil {
+		log.Fatalf("failed to get watch namespace: %v", err)
+	}
+
+	// TODO: Expose metrics port after SDK uses controller-runtime's dynamic client
+	// sdk.ExposeMetricsPort()
```

Comment:
We've updated the scaffold a little bit since #724 to include leader election and readiness probes by default.
https://github.com/operator-framework/operator-sdk/blob/master/pkg/scaffold/cmd.go#L92-L101

Can you update this `main.go` accordingly so that we are testing `test local --up-local` against what would be the current `main.go` generated for the master.
I don't believe it'll cause any issues though since the leader election util skips when it detects it's not running in the cluster.
https://github.com/operator-framework/operator-sdk/blob/master/pkg/leader/leader.go#L147-L160

Side note: we should probably update the samples memcached-operator later as well.